egs/wsj/s5/steps/nnet3/report/generate_plots.py

```diff
@@ -142,7 +142,7 @@ def GenerateAccuracyPlots(exp_dir, output_dir, plot, key = 'accuracy', file_base
     if plot:
         plt.xlabel('Iteration')
         plt.ylabel(key)
-        lgd = plt.legend(handles=plots, loc='lower center', bbox_to_anchor=(0.5, -0.2 + len(dirs) * -0.1 ), ncol=1, borderaxespad=0.)
+        lgd = plt.legend(plots, loc='lower center', bbox_to_anchor=(0.5, -0.2 + len(dirs) * -0.1 ), ncol=1, borderaxespad=0.)
```

Comment:
... I mean, excluding scripts that aren't needed and the like-- something
that we could check in.

On Fri, May 13, 2016 at 7:44 PM, Daniel Povey <dpovey@gmail.com> wrote:

> OK, I hope you can generate fairly quickly, a clean PR with just the
> changes required.
> Dan
>
>
> On Fri, May 13, 2016 at 7:22 PM, Vimal Manohar <notifications@github.com>
> wrote:
>
>> In egs/wsj/s5/steps/nnet3/report/generate_plots.py
>> <https://github.com/kaldi-asr/kaldi/pull/777#discussion_r63263678>:
>>
>> > @@ -142,7 +142,7 @@ def GenerateAccuracyPlots(exp_dir, output_dir, plot, key = 'accuracy', file_base
>> >      if plot:
>> >          plt.xlabel('Iteration')
>> >          plt.ylabel(key)
>> > -        lgd = plt.legend(handles=plots, loc='lower center', bbox_to_anchor=(0.5, -0.2 + len(dirs) * -0.1 ), ncol=1, borderaxespad=0.)
>> > +        lgd = plt.legend(plots, loc='lower center', bbox_to_anchor=(0.5, -0.2 + len(dirs) * -0.1 ), ncol=1, borderaxespad=0.)
>>
>> This is a change I made since the other one was not working atleast on
>> the CLSP cluster.
>>
>> —
>> You are receiving this because you commented.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/kaldi-asr/kaldi/pull/777/files/6d65f2c860593df154d15d5f1440ac639ac9e23d#r63263678>
>>
>
>